pkg/controller/catalog/server/controller.go

```diff
@@ -142,6 +142,10 @@ func (c *controller) CreateServiceBinding(in *scmodel.ServiceBinding) (*scmodel.
 		Parameters: in.Parameters,
 	}
 	sbr, err := client.CreateServiceBinding(to.ID, in.ID, createReq)
+	if err != nil {
+		log.Printf("Failed to create service binding: %v\n", err)
```

Comment:
A conversion from log to glog should all be done at one time. I'd prefer not to mix this error check with converting logging to glog. 

We have #168 to track.